helm_deploy/demo-app/templates/deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: django
+  labels:
+    app: django
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: django-container
```

Comment:
The name here seems a bit misleading or unclear to me - if the label is `app` the value should probably be something like 'django-reference-app`. I'd also avoid referring to "containers" in contexts like this, as you're actually referring to a Pod object, which is one or more containers.

Reply:
Okay, i'll fix that!